Final v6.3 release notes update (#33435)
mantidproject/mantid: docs/source/release/v6.3.0/mantidworkbench.rst

* Empty group workspaces can now be deleted rather than needing to be ungrouped.
* Fixed a bug in :ref:`Project Recovery<Project Recovery>` when attempting to remove non-empty directories and raising the error reporter.
* Users are no longer able to add a peak to the Fit Property Browser by clicking with the interactive tool outside of the axes (which would cause an error).
* An unhandled exeception no longer occurs when attempting to open the Fit Property Browser on a bin plot.

InstrumentViewer
----------------
New features
############

> - **In the** :ref:`Pick Tab<instrumentviewer_pick_tab>` **, a new panel allowing for direct rebinning of the workspace now exists.**

Reviewer: ```suggestion
- **In the** :ref:`Pick Tab<instrumentviewer_pick_tab>` **, a new panel allowing users to directly rebin their workspace now exists.**
```
